.editorconfig

```
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+insert_final_newline = true
+charset = utf-8
+
+[*.{ts,ts,js}]
```

`ts` is repeated twice here:

```suggestion
[*.{ts,js}]
```